src/content/docs/en/guides/astro-db.mdx

```diff
@@ -130,11 +131,13 @@ const Comment = defineTable({
 });
 ```
 
-## Seed your database
+## Seed your database for development
 
-In development, Astro will use your DB config to generate local types according to your schemas. These will be generated fresh each time the dev server is started, and will allow you to query and work with the shape of your data with type safety and autocompletion.
+In development, Astro will use your DB config to generate local types according to your schemas. These will be generated fresh from your seed file each time the dev server is started, and will allow you to query and work with the shape of your data with type safety and autocompletion.
```

Comment:
> each time the dev server is started

One subtle thing that I stumbled upon is exactly this. The dev DB is seeded every time the server starts, including live restarts.

So the entire DB is recreated and re-seeded any time you change the DB config, since that causes a restart.
But the DB is not re-seeded when you change the seed itself, since that is not used by the server runtime and as such doesn't cause a restart. Maybe it should tho 🤔

Reply:
Maybe it showld restart, but I think this goes out of the scope of the docs ahah. maybe we can modify the section like this
```suggestion
In development, Astro will use your DB config to generate local types according to your schemas. These will be generated fresh from your seed file each time the dev server is started or live reloads, and will allow you to query and work with the shape of your data with type safety and autocompletion.
```

Reply:
The text is already correct, it doesn't re-seed on live reloads. I was just pointing out that changing the seed is not a restart.

Reply:
Are we 100% sure about this behavior? I just changed one of my seeds and saved the file and `[astro:db]` "Seeded database" in the console

Here is a video demonstrating:

https://github.com/user-attachments/assets/8d4aac04-4a2b-4eac-92b5-b4c7f91be858